### DISCUSSION TITLE
Inconsistent column data: Could not convert value -inf of the field `derivatives_velMax` to type `Float`.

### DISCUSSION THREAD

**Discussion by MrVhek** [prisma/prisma#21742] [Q&A]
### Question

I have an issue with Prisma, I just migrated from an SQL usage of a postgres database. I used db pull to generate prisma schema from an existing database and now I'm trying to get data using : 
```
await prisma.binaryBlocResults.findMany({
			where: {
				id: {in: ids},
				studyid: {in: studyIds},
			},
		})
```
The schema for this table is : 
```
model BinaryBlocResults {
  id                    Int      @id(map: "binaryblocresults_pk") @unique(map: "binaryblocresults_id_uindex") @default(autoincrement())
  uid                   String?
  posX                  Json?
  posY                  Json?
  time                  Json?
  init                  Int?
  trial                 Int?
  tries                 Int?
  result                String?
  mtSpeed               Float?
  maxDeviationAbove     Float?
  maxDeviationBelow     Float?
  totalDistance         Float?
  maxVelocity           Float?
  maxAcceleration       Float?
  xPosMean              Float?
  yPosMean              Float?
  xPosSD                Float?
  yPosSD                Float?
  xEntropy              Float?
  yEntropy              Float?
  target                Json?
  xposflips             Int?
  xposreversals         Int?
  curvature_MAD         Float?
  curvature_AD          Float?
  curvature_AUC         Float?
  derivatives_totalDist Float?
  derivatives_velMax    Float?
  derivatives_accMax    Float?
  time_idle             Int?
  outlier               Int?
  studyid               Int?
  timestamp             String?
  totaltime             Int?
  score                 Float?
  rawScore              Float?
  episode               Int?
  Studies               Studies? @relation(fields: [studyid], references: [StudyId], onDelete: Cascade, onUpdate: Restrict, map: "BinaryBlocResults_Studies_null_fk")

  @@index([studyid], map: "binaryblocresults_studyid_index")
  @@index([trial], map: "binaryblocresults_trial_index")
}

```
Issue I that I have an error when trying to get the data, I have this error message : 
`Inconsistent column data: Could not convert value -inf of the field `derivatives_velMax` to type `Float`.`

So OK, but, how to debug ? My database is quite big... Can I get the ID of the problematic row somehow or is there any known issue with double precision fields in database who cannot be converted to Float prisma values ?

Thanks for your help !

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by MrVhek** [comment]
So I managed to find the culprit ids with:
```
	const idsToTest = await prisma.binaryBlocResults.findMany({select: {id: true}})
	for (const idToTest of idsToTest) {
		console.error(idToTest.id)
		await prisma.binaryBlocResults.findUnique({where: {id: idToTest.id}})
	}
```
But it seems that the -Infinity of postgres is not supported by prisma... Is there another data type than Float that manage Infinity and -Infinity in postgres ?

**Reply by ludralph** [reply]
Hi @MrVhek 

In Prisma, `-infinity` of Postgres isn't supported yet. We do have a [feature request](https://github.com/prisma/prisma/issues/3492) to provide support for it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
